hphp/runtime/test/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+enable_testing()
+find_package(GTest REQUIRED)
+include_directories(${GTEST_INCLUDE_DIRS})
+
+set(CXX_SOURCES)
+auto_sources(files "*.cpp" "RECURSE" "${CMAKE_CURRENT_SOURCE_DIR}")
+list(APPEND CXX_SOURCES ${files} )
+
+set(HEADER_SOURCES)
+auto_sources(files "*.h" "RECURSE" "${CMAKE_CURRENT_SOURCE_DIR}")
+list(APPEND HEADER_SOURCES ${files} "${CMAKE_CURRENT_SOURCE_DIR}/../../hhvm/process-init.cpp")
```

Comment:
`process-init.cpp` is an interesting name for a header file :)

Stick it in the `CXX_SOURCES` list instead.